stdlib/time.mt

```diff
@@ -0,0 +1,18 @@
+deftype Time
+  def initialize(year, month, day, hour, minute, second)
+    @year = year
+    @month = month
+    @day = day
+    @hour = hour
+    @minute = minute
+    @second = second
+  end
+
+
+  def year; @year; end
+  def month; @month; end
+  def day; @day; end
+  def hour; @hour; end
+  def minute; @minute; end
+  def second; @second; end
```

Comment:
Setters for these properties would be great :)

Reply:
I am not sure, should we be able to change the values of a Time type once instantiated?  I don't believe Ruby or Crystal allows that

Reply:
Hmmm, you're right. I'm not sure where I got the impression that they were mutable, but for now leaving them immutable seems like the right call.

Maybe (for my own sake, cause I'm gonna forget again) add a comment above these saying setters are purposefully omitted? :)